### DISCUSSION TITLE
Same runner on 2 VM's

### DISCUSSION THREAD

**Discussion by shaibrown5** [community/community#27066] [Actions]
<p>I’m trying to create a pipeline that is triggered with github actions. each action spins an azure VM and runs on self hosted. for this I have an image with a single github runner already installed on it.</p>
<p>can the same runner installed on different vm’s run different workflows simultaneously? if not , is there a solution that might work</p>

**Comment by joshmgross** [comment, marked as answer]
<p>:wave: Hey <a class="mention" href="/shaibrown5">@shaibrown5</a>,</p>
<p>A single runner can’t run different workflows simultaneously. I recommend configuring each VM as a separate runner.</p>
<p>See <a href="https://docs.github.com/en/actions/hosting-your-own-runners/adding-self-hosted-runners" class="inline-onebox">Adding self-hosted runners - GitHub Docs</a> for more info.</p>
